etc/sdam_viz.js

```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/env node
-'use strict';
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-var-requires */
 
```

Comment:
Did you mean to move this file into the `etc` directory?  I don't see this directory in the codebase as it is now, so I'd be curious to know what the purpose of this directory is if this change was intentional

Reply:
Yes I did, it should be there, there's a prepare.js script and crawfish.mjs in there. Since this is a stand alone helper script and doesn't get imported by any tests it would better belong in here.  Also the imports were broken